common/src/main/java/com/hartwig/actin/molecular/datamodel/driver/VirusType.java

```diff
@@ -6,5 +6,22 @@ public enum VirusType {
     HUMAN_PAPILLOMA_VIRUS,
     HEPATITIS_B_VIRUS,
     HUMAN_HERPES_VIRUS_8,
-    OTHER
+    OTHER;
+
+    public String toString() {
+        switch (this) {
+            case MERKEL_CELL_VIRUS:
+                return "MCV";
+            case EPSTEIN_BARR_VIRUS:
+                return "EBV";
+            case HUMAN_PAPILLOMA_VIRUS:
+                return "HPV";
+            case HEPATITIS_B_VIRUS:
+                return "HBV";
+            case HUMAN_HERPES_VIRUS_8:
+                return "HHV8";
+            default:
+                return "Other";
+        }
```

Comment:
Doesn't this potentially impact serialisation / deserialization? I generally use "display" for the name of the function that generates a string that is meant for display to a human being

Reply:
It is somewhat inconsistent with how I've done this in the past (not sure how many of such cases exist in ACTIN, the majority probably in serve or hmftools), but I get the point. One advantage with explicit "display" function is that you can't remove it without breaking code. With toString this is less clear.